plugins/base/utils/source-setting.cpp

```diff
@@ -113,19 +113,22 @@ void SetSourceSetting(obs_source_t *source, const SourceSetting &setting,
 				obs_data_set_int(data, id.c_str(), *intValue);
 				break;
 			}
+
 			auto doubleValue = GetDouble(value);
 			if (doubleValue.has_value()) {
-				obs_data_set_int(data, id.c_str(),
```

Comment:
I think this needs to remain `obs_data_set_int` doesn't it?

Reply:
Right, since the function accept `long long` it makes more sense to use that, I'll revert to that.